docs/symphony-book/cli/build_cli.md

```diff
@@ -80,7 +80,7 @@ _(last edit: 10/24/2023)_
 1. Check `symphony/cli/install/install.sh` and `symphony/cli/install/install.ps1` to a public repository. Users will be instructed to use scripts from this repo for the one-command experience, for example:
 
    ```bash
-   wget -q https://raw.githubusercontent.com/eclipse-symphony/symphony/master/cli/install/install.sh -O - | /bin/bash
+   wget -q https://raw.githubusercontent.com/Haishi2016/Vault818/master/cli/install/install.sh -O - | /bin/bash
```

Comment:
Please run a pass and fix all URLs

Reply:
Sorry about these -- a mistake when porting over my changes from the old repo. I'm scrubbing them all now.